src/main/java/eu/lucaventuri/common/Exceptions.java

```diff
@@ -147,19 +151,15 @@ public static void logShort(RunnableEx run) {
         try {
             run.run();
         } catch (Throwable t) {
-            // log.error(t.getMessage(), t); Not working...
-            // FIXME: log for real
-            System.err.println(t);
+            logger.log(Level.FINEST, t.getMessage(), t);
```

Comment:
Many of the statements on this class are wrong, and this is critical as this class is used a lot to decide how to print the exceptions:
- FINEST (except if the method is called silence or ignore) should probably be INFO: we want to see those message
- If a method is called xxxShort(), it means I do not want to see the stack trace, just the message and the type of the exception